utilities/convert_frosst/smirnoffishFrcmod.parm99Frosst.txt

```diff
@@ -49,6 +50,17 @@ NONBON
 [#17:1]          1.948  0.265   parm99 Cl Fox, JPCB,102,8070,(98),flex.mdl CHCl3
 [#35:1]           2.22  0.320   parm99 Br Junmei(?)
 [#53:1]           2.35  0.40    parm99 I JCC,7,(1986),230;
+# Monovalent ions from parm99
+[#3:1]          1.1370  0.0183   parm99 
+[#11:1]         1.8680  0.00277  parm99 
+[#19:1]         2.6580  0.000328 parm99
+[#37:1]         2.9560  0.00017  parm99
+[#55:1]         3.3950  0.0000806 parm99
+[#9X0:1]        1.75    0.061    parm99
+[#17X0:1]       1.948   0.265    parm99
+[#35X0:1]       2.22    0.320    parm99
+[#53X0:1]       2.35    0.40     parm99  
```

Comment:
Adding formal charges to the SMIRKS might make it easier to understand what these are. Though, they aren't necessary for actually assigning the parameters correctly.